test/martha/forceAccessUrl.test.js

```diff
@@ -0,0 +1,68 @@
+const test = require('ava');
+const sinon = require('sinon');
+const config = require('../../common/config');
+const helpers = require('./_helpers.js')
+
+const { marthaV3Handler: marthaV3 } = require('../../martha/martha_v3');
+const apiAdapter = require('../../common/api_adapter');
+
+const terraAuth = "Bearer XYZ_TerraAuthToken";
+const bucketId = "bucket_id";
+const responseId = `v1_ResponseId_${bucketId}`;
+const gsAccessId = "gcp-us-central1";
+const gsUrl = `gs://broad-jade-dev-data-bucket/BucketIdPrefix/${bucketId}`;
+const drsUri = `drs://${config.HOST_TDR_DEV}/${responseId}`
+
+
+const jadeAccessUrlMetadataResponse = {
+  "id": `${responseId}`,
+  "access_methods": [
+    {
+      "type": "gs",
+      "access_id": `${gsAccessId}`,
+    },
+  ],
+};
+
+
+test.serial.beforeEach(() => {
```

Comment:
I'm not enough of an expert with `ava` (the test runner) to know if defining this here will affect tests outside of this file. But it does seem like each semantically-related group of tests should be able to have their own set of setup/teardown functionality.